caesars-palace/blocks/carousel/carousel.js

```diff
@@ -283,9 +289,11 @@ export default async function decorate(block) {
   // add carousel to page
   const slides = [...block.children];
   maxVisibleSlides = slides.length;
+  const slidesToAdd = new Array(maxVisibleSlides);
```

Comment:
For my own learning, what's the benefit of this change?

Reply:
Instead of multiple DOM manipulations (one in each loop below), I do only one after the loop to add all slides at once.